inference-engine/src/transformations/src/transformations/op_conversions/convert_interpolate1_to_interpolate4.cpp

```diff
@@ -16,56 +16,62 @@
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertInterpolate1ToInterpolate4, "ConvertInterpolate1ToInterpolate4", 0);
 
 ngraph::pass::ConvertInterpolate1ToInterpolate4::ConvertInterpolate1ToInterpolate4() {
-    MATCHER_SCOPE(ConvertInterpolate1ToInterpolate4);
```

Comment:
Why did you remove `MATCHER_SCOPE` macro?!

Reply:
To apply the comment of @GlebKazantaev.

Reply:
I couldn't find the comment from @GlebKazantaev where he asks to remove this macro.
Please revert it back. This macro is needed for CC feature.

Reply:
Fixed.